### ISSUE TITLE
[BUG/ISSUE] WingetUI resouce consumption when minimized.

### ISSUE THREAD

**Issue by akgis32** [marticliment/WingetUI#728]
## Before submitting the issue

- [x] **I have checked that the bug is not a duplicate or, if it is, it needs to be reopened**
- [x] **I have checked that I am running the latest version of WingetUI**
- [x] **I have tried resetting and reinstalling WingetUI and the issue persisted or happened again**
- [x] **I have checked the [FAQ](https://github.com/martinet101/WingetUI#faq) section for solutions**
- [x] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#issues) and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**

## Describe the bug
WingetUI keeps consuming resouces when "closed" to tray, arround 40mil to 80mil CPU cycles delta and 3 KB/s of IO, update checker is off and notifications. Only Winget Sources are active

## To Reproduce
Steps to reproduce the behaviour:
Open Winget
Close it to tray
The resouces being consumed can be observed on Sysinternals Process Explorer

## Expected behaviour
When minimized resouce consumption should be close to zero, expecialy if the update checker is off

## The Log

<details>
Logging doesnt show revelant info for this

**Comment by marticliment** [comment]
Hi, i'm sorry, what does 40mil mean?

**Comment by rumplin** [comment]
> Sysinternals Process Explorer

the percentage of CPU time in the last second (or whatever the update speed is set to in Sysinternals Process Explorer)

I experienced high resource usage on a low spec laptop recently I was doing a fresh install of Windows 10.
I found it very slow, especially when the blur was applied, but the inital ticket poster is talking abount resource usage in general of the app.
Maybe worth checking some methods that are run in an interval?

**Comment by rumplin** [comment]
Well I recently had the chance to work on the following machines:
Intel Core 2 Duo P8400, 1GB RAM
AMD E1-2100 APU, 4GB RAM

This is super legacy, so don't worry about that. 
But the author of the ticket was surely using WingeUI on a newer system.

**Comment by marticliment** [comment]
I don't know about OP's system, but you can't expect lightning performance from. any app given the specs said.

and for the transparencies, you can disable them from windows settings.

**Comment by ppvnf** [comment]
I think reducing CPU consumption would be great, but this issue is pointless unless there are suggestions explaining how to do so

**Comment by marticliment** [comment]
Yes. Optimisations are always made. But note that an application that  handles 14k+ packages requires come cpu computing power, because there are lots of data to parse, organize and show to the user
